rllib/offline/offline_data.py

```diff
@@ -77,6 +77,7 @@ def sample(
                 fn_constructor_kwargs={
                     "config": self.config,
                     "learner": self.learner_handles[0],
+                    "spaces": self.spaces["__env__"],
```

Comment:
nit: Can we merge the other (`__env__` constant) PR into this one and then push again. This way, we won't forget to change this here.